### ISSUE TITLE
Regarding image extraction

### ISSUE THREAD

**Issue by Ashwin-Ramesh2607** [maudzung/TTNet-Real-time-Analysis-System-for-Table-Tennis-Pytorch#6]
Hey, great repo! I have question regarding the image dataset extraction since I am not really focused on the event detection part. I have seen that have recommended to execute the `extract_smooth_labellings.py` to select the required images. I understand that if I need event detection. However, if I dont want to train on event detection, should I still go with `extract_smooth_labellings.py` or use `extract_all_images.py` instead, so that I have more training data for the segmentation and ball detection tasks?

**Comment by maudzung** [comment]
Hi @Ashwin-Ramesh2607 
In the TTNet dataset, the ball positions and segmentations of the human, table, and scoreboard are auto-annotated based on key event frames. Hence, not every frame has labels for ball position and segmentation. Although you don't focus on the event detection task, you also need to execute `python extract_smooth_labellings.py` to save your storage and your time.

**Comment by Ashwin-Ramesh2607** [comment]
@maudzung Alright got it, so to confirm. Even if I am not concerned with event detection, I should use `extract_smooth_labellings.py` correct?

**Comment by maudzung** [comment]
Yes. You should extract images from videos by using extract_smooth_labelling.py script. 

You can read more about the data preparation in the TTNet paper. As they mentioned, only the events were fully annotated.

**Comment by Ashwin-Ramesh2607** [comment]
Okay thanks, understood. I noticed in the README that you have implemented a new model for this same dataset. Currently the repo is not public. Do you happen to know any tentative date when it will be available for public use?

**Comment by maudzung** [comment]
Hi @Ashwin-Ramesh2607 
I'm trying to improve the accuracy of event spottings. For the segmentation and ball detection tasks, I think the model is working well at this time, it surpasses the TTNet model's performance.
